app/assets/stylesheets/pages/dashboard.scss

```diff
@@ -1,3 +1,33 @@
-// Place all the styles related to the dashboard controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+// Place all the styles related to the dashboard controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.wrappr{

```

Comment:
For example you could use dashboard-wrapper instead of wrappr. Do something similar for all the classes defined in this file.